pkg/storage/module/delete_test.go

```diff
@@ -0,0 +1,55 @@
+package module
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/gobuffalo/envy"
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/suite"
+)
+
+type DeleteTests struct {
+	suite.Suite
+}
+
+func TestDelete(t *testing.T) {
+	suite.Run(t, new(DeleteTests))
+}
+
+func (m *DeleteTests) TestDeleteTimeout() {
+	r := m.Require()
+	envy.Set("ATHENS_TIMEOUT", "1")
+	defer envy.Set("ATHENS_TIMEOUT", "300")
```

Comment:
I'm wondering about the use of these defer for the environment variable. What was the reasoning? It seems like each test uses a timeout of 1 second and sets it back to 300. If you are trying to keep the original value for other tests, you can instead use `(m *DeleteTests) BeforeTest()` and `(m *DeleteTests) AfterTest()` .